### ISSUE TITLE
[gear] We need gear!

### ISSUE THREAD

**Issue by Qard** [dancejs/Vancouver#1]
As this is the first event, I'm pretty lacking in gear presently. I'm going to be buying/borrowing a whole bunch of stuff. If you have anything you are willing to part with temporarily or permanently, let me know.

Needed gear:
- [ ] Mixer
- [ ] PA speakers
  - [ ] Subs?
- [ ] Amp
- [ ] Compressor?
- [ ] Mics (wireless?)
- [ ] ALL THE CABLES!
  - [ ] Extension cables (2, probably)
  - [ ] Lots of XLR? Depends on amp -> speaker, I guess.
  - [x] 1/8 -> 1/8? (Have some already)
  - [x] 1/4 -> 1/4? (Have some already)
  - [x] 1/8 -> RCA? (Have some already)
- [ ] Lighting? (what do we want?)

Depending on availability of stuff, I may just get the same gear the Oakland group has. Going to take a closer look at the gear situation soon.

**Comment by Qard** [comment]
I've started an [Amazon wishlist](http://www.amazon.ca/registry/wishlist/2NSGKAYZJSO9G). Everything on there is stuff that I'm at least mostly sure about.

I need to think a bit more about speakers. I'm not yet sure whether to go powered or unpowered. I'm leaning toward unpowered as powered is a little pricier and requires more cables.

Thoughts? Suggestions for other stuff? @othiym23 @bcoe

**Comment by othiym23** [comment]
I personally like having individual, dedicated components:
- easier to replace and upgrade the sound system piecewise as needs change
- more likely to be serviceable if it's not just one compact wad of electronics
- easier to design a system that fits your needs

I know that @bcoe is a big fan of Mackie's powered speakers, but I've always had good experiences with [JBL's pro sound reinforcement](http://www.jblpro.com/www/products/portable-market/stx800-series/stx835) speakers. The big advantage of a powered speaker setup is that they're much easier to move around, which is a big deal if you ever think you might want to do outdoor and / or renegade events.

If you do decide to get unpowered speakers, be sure to buy more amplification than you think you need. It is extremely hard to find data sheets for solid state amps that specify output in terms of sustained RMS rather than peak output, so a "4000 watt" amp from most vendors is not going to be capable of cranking out lots of bass of time without overheating.

That mixer has worked OK for Oakland DanceJS, but it's really easy to run out of outputs, especially if anybody wants a line out to record talks. Something that has at least 2 balanced / XLR pairs out would not go amiss, and if you've got the time and patience to figure it out, a full-on patch bay could be very useful.

Also get a rack and rails to mount everything in the rack. That way you can wire up the mixer, amps, and other outboard signal chain gear once, and then just set it up at the venue, instead of having to rewire everything every tie.

**Comment by Qard** [comment]
Good point about simple components. My home set up uses powered studio monitors, but I still need a preamp into it.

I'm definitely considering a rack of some sort, if I end up buying a bunch of my own gear. We'll see if the budget allows it.

What do you think of the Mackie mixer in the wishlist? It seems to have L/R outs for XLR, 1/8 and RCA. In my brief look through Amazon, I haven't found any mixer with more than one XLR pair. Do you know of any off-hand that'd be good to check out?

**Comment by bcoe** [comment]
@Qard for what it's worth, here's an inventory of some of the equipment we use:

https://github.com/nexxy/ravejs

Having said this, we haven't quite got the subwoofer situation under-control yet ... so it's probably worth rethinking some of the components.

**Comment by Qard** [comment]
https://www.long-mcquade.com/ has reasonable rental prices.

**Comment by Qard** [comment]
Looks like the first event is happening at a venue with house gear, so we can skip finding gear this time.
